src/map/utils/battleutils.h

```diff
@@ -44,22 +44,22 @@ enum ENSPELL
 {
     ENSPELL_NONE = 0,
     ENSPELL_I_FIRE = 1,
-    ENSPELL_I_EARTH = 2,
-    ENSPELL_I_WATER = 3,
-    ENSPELL_I_WIND = 4,
-    ENSPELL_I_ICE = 5,
-    ENSPELL_I_THUNDER = 6,
+    ENSPELL_I_ICE = 2,
+    ENSPELL_I_WIND = 3,
+    ENSPELL_I_EARTH = 4,
+    ENSPELL_I_THUNDER = 5,
+    ENSPELL_I_WATER = 6,
     ENSPELL_I_LIGHT = 7,
     ENSPELL_I_DARK = 8,
     ENSPELL_II_FIRE = 9,
-    ENSPELL_II_EARTH = 10,
-    ENSPELL_II_WATER = 11,
-    ENSPELL_II_WIND = 12,
-    ENSPELL_II_ICE = 13,
-    ENSPELL_II_THUNDER = 14,
+    ENSPELL_II_ICE = 10,
+    ENSPELL_II_WIND = 11,
+    ENSPELL_II_EARTH = 12,
+    ENSPELL_II_THUNDER = 13,
+    ENSPELL_II_WATER = 14,
     ENSPELL_II_LIGHT = 15,
-    ENSPELL_BLOOD_WEAPON = 16,
-    ENSPELL_ROLLING_THUNDER = 17,
+    ENSPELL_II_DARK = 16,
+    ENSPELL_BLOOD_WEAPON = 17,
     ENSPELL_AUSPICE = 18,
```

Comment:
[Endark II](https://www.bg-wiki.com/bg/Endark_II) wasn't in this table.

"Enspell Rolling Thunder" is never used - the [ability simply uses Enthunder instead](https://github.com/ibm2431/topaz/blob/master/scripts/globals/abilities/pets/rolling_thunder.lua).

So Blood Weapon's "enspell ID" got pushed back by one.